src/chrome/internal/stackTraces/codeFlowStackTrace.ts

```diff
@@ -0,0 +1,12 @@
+import { CodeFlowFrame } from './callFrame';
+import { IScript } from '../scripts/script';
+
+/** This class represents a stack trace that only has code flow information, but no state
```

Comment:
Another minor style point, I think the comment should start on the line after `/**`, e.g.

```
/**
 * comment
 */

Reply:
Changed